bumble/hci.py

```diff
@@ -3738,39 +3757,15 @@ class HCI_LE_Read_Advertising_Physical_Channel_Tx_Power_Command(HCI_Command):
 
 
 # -----------------------------------------------------------------------------
-@HCI_Command.command(
-    [
-        (
-            'advertising_data',
-            {
-                'parser': HCI_Object.parse_length_prefixed_bytes,
-                'serializer': functools.partial(
-                    HCI_Object.serialize_length_prefixed_bytes, padded_size=32
-                ),
-            },
-        )
-    ]
-)
+@HCI_Command.command([('advertising_data', 'v')])
```

Comment:
This won't pad to 32 bytes if `advertising_data` is less than 32 bytes.

Reply:
Done. Reverted.